infra/common/src/main/java/kg/apc/jmeter/JMeterPluginsUtils.java

```diff
@@ -179,8 +179,12 @@ public static Component addHelpLinkToPanel(Container panel, String helpPage) {
             return panel;
         }
 
-        if (helpPage.endsWith("Gui")) {
-            helpPage = helpPage.substring(0, helpPage.length() - 3);
+        // build link to help page unless it's already a URL:
+        if (!helpPage.toLowerCase().startsWith("http")) {
```

Comment:
That will match any page starting with HTTP. You should use "http:// or https://" logic instead